pkgs/development/python-modules/pdfx/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub, pdfminer, chardet, pytest }:
+
+buildPythonPackage rec {
+  pname = "pdfx";
+  version = "1.3.1";
+
+  src = fetchFromGitHub {
+    owner = "metachris";
+    repo = "pdfx";
+    rev = "v${version}";
+    sha256 = "1183k4h5qdf8y0imbir9ja3yzzsvdmqgbv3bi6dnkgr1wy2xfr0v";
+  };
+
+  prePatch = ''
+    sed -i -e "s|pdfminer2|pdfminer.six|" setup.py
```

Comment:
I think you should package pdfminer2 instead of using an old version.

Reply:
Wasn't `pdfminer.six` explicitly replaced in https://github.com/metachris/pdfx/commit/cee76e10520ba5a36386dbf5eb43c8d9f3243e59#diff-2eeaed663bd0d25b7e608891384b7298?

Reply:
> Wasn't pdfminer.six explicitly replaced in metachris/pdfx@cee76e1?

I didn't knew that, but seems that there is no significant changes in pdfminer2 https://github.com/pdfminer/pdfminer.six/compare/master...metachris:master.
pdfx works with pdfminer.six, but i can package pdfminer2 if you prefer it

Reply:
Let's first wait for the maintainers answer to https://github.com/metachris/pdfx/pull/28.

Reply:
As the package seems unmaintained, I'd say we just go with your change. However, please add a link to your PR as a comment.